src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml

```diff
@@ -37,7 +36,9 @@
             <argument type="service" id="session.storage.metadata_bag" />
         </service>
 
-        <service id="session.flash_bag" class="Symfony\Component\HttpFoundation\Session\Flash\FlashBag" />
+        <service id="session.flash_bag" class="Symfony\Component\HttpFoundation\Session\Flash\FlashBag">
+            <factory service="session" method="getFlashBag"/>
+        </service>
         <service id="Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface" alias="session.flash_bag" />
 
         <service id="session.attribute_bag" class="Symfony\Component\HttpFoundation\Session\Attribute\AttributeBag" />
```

Comment:
@nicolas-grekas should we also configure session.attribute_bag in the same way for consistency?

Reply:
yes that'd make sense

Reply:
Then on master, we could deprecate these two services.

Reply:
@warslett up to send a PR to deprecate these?

Reply:
@nicolas-grekas here: https://github.com/symfony/symfony/pull/36273 I get build failures but doesn't seem related. Are the builds on master wonky right now?